modules/virt-configuring-downward-metrics.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-downward-metrics_virt-using-downward-metrics"]
+= Configuring downward metrics
+
+You enable the capturing of downward metrics for a host VM by creating a configuration file that includes the following options:
+
+* A `downwardMetrics` device
+* A `virtio-serial` port, through which the metrics are exposed
+
+You must also enable a `DownwardMetrics` feature gate.
+
+.Prerequisites
+* Administrator privileges are required to enable the feature gate.
+
+.Procedure
+
+. Edit or create a YAML file to expose the metrics by including a `downwardMetrics` device, a `virtio-serial` port, and the `DownwardMetrics` feature gate as shown in the following example:
++
+.Example downwardMetrics configuration file
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
```

Comment:
@jcanocan Can you please provide an example VM, no VMI, using instancetypes?

Reply:
Sure! This is a complete and working VM definition using instancetypes:

```yaml
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata:
  name: fedora
  namespace: default
spec:
  dataVolumeTemplates:
    - metadata:
        name: fedora-volume
      spec:
        sourceRef:
          kind: DataSource
          name: fedora
          namespace: openshift-virtualization-os-images
        storage:
          resources: {}
          storageClassName: hostpath-csi-basic
  instancetype:
    name: u1.medium
  preference:
    name: fedora
  running: true
  template:
    metadata:
      labels:
        app.kubernetes.io/name: headless
    spec:
      domain:
        devices:
          downwardMetrics: {}
      subdomain: headless
      volumes:
        - dataVolume:
            name: fedora-volume
          name: rootdisk
        - cloudInitNoCloud:
            userData: |
              #cloud-config
              chpasswd:
                expire: false
              password: nye8-9j68-qr57
              user: fedora
          name: cloudinitdisk
```